x-template.js

```diff
@@ -780,18 +633,8 @@ class TemplateEngine {
     }
   }
 
-  static #throwUpdaterError(updater, binding) {
```

Comment:
We’re down to _just one_ “updater”, which is `ifDefined`. By `2.x` we will remove this as well an expect that authors will leverage the `??` binding to get the same effect.